elements/pfe-modal/src/pfe-modal.scss

```diff
@@ -56,21 +67,33 @@ $LOCAL-VARIABLES: (
 	}
 	&__window {
 		--context: #{pfe-local(context)};
+
+		//@todo , this is a workaround for issue 1090
+		//@include pfe-set-broadcast-theme(light);
 		@include pfe-set-broadcast-context(#{pfe-fetch(context)});
 
 		position: relative;
+		top: pfe-local(Top);
 		max-width: pfe-local(MaxWidth--mobile);
 		min-width: pfe-local(MinWidth);
 		max-height: pfe-local(MaxHeight);
 		margin: 0 auto;
 		box-shadow: pfe-var(box-shadow--lg);
-		background-color: pfe-var(surface--lightest);
+		background-color: pfe-local(BackgroundColor, $region: window); 
+
 		color: pfe-var(text);
 		border-radius: pfe-var(ui--border-radius);
 		
 		@media screen and (min-width: $pfe-modal--breakpoint--medium) {
 			max-width: pfe-local(MaxWidth);
 		}
+		:host([pfe-media]) & { 
+			--pfe-modal--MinWidth: none;
```

Comment:
Can you update this to redefine the properties so that the variables can stay empty?